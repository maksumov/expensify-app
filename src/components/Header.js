import React from 'react'
import { NavLink } from 'react-router-dom'

const Header = () => (
  <header>
    <h1>Expensify</h1>
    <div>
      <NavLink exact to='/' activeClassName='is-active'>
        Go home
      </NavLink>
      {' | '}
      <NavLink to='/create' activeClassName='is-active'>
        Add Expense
      </NavLink>
      {' | '}
      <NavLink to='/help' activeClassName='is-active'>
        Help
      </NavLink>
    </div>
  </header>
)

export default Header
