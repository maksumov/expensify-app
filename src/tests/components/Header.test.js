import React from 'react'
import ReactShallowRenderer from 'react-test-renderer/shallow'
import Header from '../../components/Header'

// react-test-renderer
test('should render Header correctly', () => {
  const renderer = new ReactShallowRenderer()
  renderer.render(<Header />)
  expect(renderer.getRenderOutput()).toMatchSnapshot()
})
