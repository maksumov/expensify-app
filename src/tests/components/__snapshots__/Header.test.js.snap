// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <div>
    <NavLink
      activeClassName="is-active"
      exact={true}
      to="/dashboard"
    >
      Dashboard
    </NavLink>
     | 
    <NavLink
      activeClassName="is-active"
      to="/create"
    >
      Add Expense
    </NavLink>
     | 
    <NavLink
      activeClassName="is-active"
      to="/help"
    >
      Help
    </NavLink>
    <button
      onClick={[MockFunction]}
    >
      Logout
    </button>
  </div>
</header>
`;
